build/debian-base/Makefile

```diff
@@ -18,7 +18,7 @@ REGISTRY ?= staging-k8s.gcr.io
 IMAGE ?= $(REGISTRY)/debian-base
 BUILD_IMAGE ?= debian-build
 
-TAG ?= 0.4.1
+TAG ?= v1.0.0
```

Comment:
Should this be 1.0.0 (No `v`)?
https://pantheon.corp.google.com/gcr/images/google-containers/GLOBAL/debian-base@sha256:d292fd380195fb09fa4e2f1d3303158edd7cd5e4843c3cabc99ac0494f1329fd/details?tab=info